docs/markdown/android/searchguide.md

```diff
@@ -0,0 +1,97 @@
+---
+title: SearchGuide
+description: A SearchGuide appends and refines a search query. They appear under [SearchField](/android/searchfield) after user submits a search input.
+fullwidth: true
+---
+
+<ImgContainer src="https://www.pinterest-assets.com/AssetLink/6cqbx311pg433xk57bo866233s36gn13/header-searchguide-and-png.png" alt="An example of a default search guide stream."/>
+
+## Usage guidelines
+
+<TwoCol>
+<Group>
+<Do title="When to use" />
+- To refine the original search query with more accurate and targeted keywords
+
+</Group>
+<Group>
+<Dont title="When not to use" />
+- To recommend related content or descriptions. Use [Tag](web/tag) instead
+- To switch between different, yet related views. Use [Tabs](/android/tabs) instead
+- To start a new search based solely on the content of the SearchGuide
+</Group>
+</TwoCol>
+
+## Best practices
+
+- SearchGuides are serial and appear in multiples. When a query doesn’t produce enough guides of sufficient quality, they shouldn’t be shown
+- Limit the number of SearchGuides to 9 per search query. If there are less number of SearchGuides present for a give query then by default we show that number. Limiting vertical scrolling helps to enhance the user experience by reducing the effort required to navigate and locate content
```

Comment:
grammar/typos:
- "If there are _fewer_ SearchGuides present…"
- "for a _given_ query, then by default…" (note the added comma)